tests/PermissionMiddlewareTest.php

```diff
@@ -75,6 +77,10 @@ public function a_user_cannot_access_a_route_protected_by_the_permission_middlew
     /** @test */
     public function a_client_cannot_access_a_route_protected_by_the_permission_middleware_of_a_different_guard(): void
     {
+        if ($this->getLaravelVersion() < 9) {
+            $this->markTestSkipped('must be laravel 9 or earlier.');
```

Comment:
Shouldn't this be `must be laravel 9 or newer/ later`?

Reply:
Yes a typo, but what I was interested in knowing is if it meets all the scenarios you describe